src/Cake.Common/Tools/DotNetCore/Test/DotNetCoreTester.cs

```diff
@@ -99,6 +99,13 @@ private ProcessArgumentBuilder GetArguments(string project, DotNetCoreTestSettin
                 builder.Append("--no-build");
             }
 
+            // Xml output for test results
```

Comment:
@cake-build/team wonder if we should require framework to be set? Otherwise only last tested framework will be in report right?